tests/Feature/Http/Controllers/Staff/ChatRoomControllerTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Tests\Feature\Http\Controllers\Staff;
+
+use App\Models\Chatroom;
+use App\Models\Group;
+use App\Models\User;
+use GroupsTableSeeder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+/**
+ * @see \App\Http\Controllers\Staff\ChatRoomController
+ */
+class ChatRoomControllerTest extends TestCase
+{
+    use RefreshDatabase;
```

Comment:
I suggest omitting this trait here because we use it in the parent class, `TestCase`. I say this specifically because if we ever need to disable the trait while debugging tests, it's a lot easier to do it in one place (the parent class) than have to hunt-down all the child classes that might use it.